roles/remediate/tasks/leapp_relative_symlinks.yml

```diff
@@ -1,54 +1,38 @@
+      map(attribute='detail.remediations') | flatten | list }}"
+    remediation: "{{ remediation_matches | selectattr('type', 'eq', 'command') | first
+      if remediation_matches | length > 0 else {} }}"
   block:
-    - name: leapp_relative_symlinks | Check that the leapp-report.json with remediation command exists
-      ansible.builtin.stat:
-        path: "{{ leapp_report_location }}"
-      register: leapp_report_stat
-
-    - name: leapp_relative_symlinks | End execution of playbook if leapp report does not exist (not possible to remediate)
-      ansible.builtin.set_fact:
-        leapp_report_missing: true
-      when: leapp_report_stat.stat.exists is false
-      failed_when: leapp_report_stat.stat.exists is false
-
-    - name: leapp_relative_symlinks | Read leapp report
-      ansible.builtin.slurp:
-        src: "{{ leapp_report_location }}"
-      register: leappreport
-
-    - name: leapp_relative_symlinks | Parse leapp report to json
-      ansible.builtin.set_fact:
-        leappreportdata: "{{ leappreport.content | b64decode | from_json }}"
-
-    - name: leapp_relative_symlinks | Find matching entries
-      ansible.builtin.set_fact:
-        remediation: "{{ item.detail.remediations | selectattr('type', 'eq', 'command') | first }}"
-      loop: "{{ leappreportdata.entries }}"
-      when: item.title == entry_title and (item.detail.remediations | selectattr('type', 'eq', 'command') | list | length > 0)
-
     - name: leapp_relative_symlinks | End execution of playbook if no entry found in leapp report
-      ansible.builtin.set_fact:
-        leapp_report_missing: true
-      failed_when: remediation is not defined
-
-    - name: leapp_relative_symlinks | Output command to be executed
-      ansible.builtin.debug:
-        msg: "{{ remediation.context | last }}"
-
-    - name: leapp_relative_symlinks | Set links in root directory to be relative
-      ansible.builtin.command: "{{ remediation.context | last }}"
-      when: remediation is defined
-      register: leapp_relative_symlinks
-      changed_when: leapp_relative_symlinks.rc == 0
-
-  rescue:
-    - name: leapp_relative_symlinks | Continue when leapp report is missing
       ansible.builtin.debug:
-        msg: "Leapp report missing or did not contain any matches. Skipping this task."
-      when: leapp_report_missing is defined and leapp_report_missing is true
+        msg: "No matching entry found in leapp report. Skipping this task."
+      when: remediation | length == 0
+
+    - name: leapp_relative_symlinks | Process remediation
+      when: remediation | length > 0
+      block:
+        - name: leapp_relative_symlinks | Output command to be executed
+          ansible.builtin.debug:
+            msg: "{{ remediation.context }}"
+
+        # The old version passed the list directly to the command.  I didn't think
+        # this was possible.  Guard against something weird here.  Assume non-string
+        # is a list.
```

Comment:
@PeterMocary do you know where leapp inhibitors are stored? Maybe there is some KCS about this inhibitor that can provide us more info